src/dict.h

```diff
@@ -87,6 +87,8 @@ typedef struct dictType {
     /* If embedded_entry flag is set, it indicates that a copy of the key is created and the key is embedded
      * as part of the dict entry. */
     unsigned int embedded_entry : 1;
+    /* Perform rehashing during resizing instead of deferring it across multiple steps */
+    unsigned int no_deferred_rehash : 1;
```

Comment:
Not sure about the terminology here, but I would call this "incremental rehashing", `no_incremental_rehash`.

Reply:
Thanks, Done.